requirements.txt

```
@@ -2,5 +2,5 @@ django>=2.2,<3.2
 django-leaflet>=0.23,<0.28
 channels>=2.3.0,<2.5.0
 Pillow>=6.2.0,<=8.1.0
-openwisp-utils~=0.6.0
+openwisp-utils>=0.6.0,<=0.7.0
```

```suggestion
openwisp-utils[qa]>=0.6.0,<0.8
```
This way will support all minor versions of openwisp-utils 0.7